### PR TITLE
Add CI and generate an exe

### PR DESCRIPTION
**Pull request by addy419**
Hi @TheAndreLuiz, I've added a CI workflow that you can add to this repo and check if it compiles successfully. I've been trying to compile NovlusDashboard on Windows, Linux and through this CI but had no luck so far.

Hope this PR helps you in identifying what's missing upstream so that we have a working executable.

### CODE REVIEW

**Comment by addy419** [comment]
@TheAndreLuiz Looks like it's missing all the Syncfusion packages:

```
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2413,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "Syncfusion.Compression.Base, Version=20.2460.0.36, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Core\Vcc.Nolvus.Core.csproj]
          For SearchPath "{TargetFrameworkDirectory}".
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Syncfusion.Compression.Base.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Facades\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Facades\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Facades\Syncfusion.Compression.Base.exe", but it didn't exist.
          For SearchPath "{AssemblyFoldersFromConfig:C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\AssemblyFolders.config,v4.8}".
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\SqlDb\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\SqlDb\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\SqlDb\Syncfusion.Compression.Base.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\PublicAssemblies\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\PublicAssemblies\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\PublicAssemblies\Syncfusion.Compression.Base.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.5\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.5\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.5\Syncfusion.Compression.Base.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.0\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.0\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.0\Syncfusion.Compression.Base.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v2.0\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v2.0\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v2.0\Syncfusion.Compression.Base.exe", but it didn't exist.
          For SearchPath "{Registry:Software\Microsoft\.NETFramework,v4.8,AssemblyFoldersEx}".
          Considered AssemblyFoldersEx locations.
          For SearchPath "{AssemblyFolders}".
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\Syncfusion.Compression.Base.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\Syncfusion.Compression.Base.exe", but it didn't exist.
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\Syncfusion.Compression.Base.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\Syncfusion.Compression.Base.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\WiX Toolset v3.14\SDK\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\WiX Toolset v3.14\SDK\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\WiX Toolset v3.14\SDK\Syncfusion.Compression.Base.exe", but it didn't exist.
          For SearchPath "{GAC}".
          Considered "Syncfusion.Compression.Base, Version=20.2460.0.36, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL", which was not found in the GAC.
          For SearchPath "{RawFileName}".
          Considered treating "Syncfusion.Compression.Base, Version=20.2460.0.36, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL" as a file name, but it didn't exist.
          For SearchPath "..\Assemblies\".
          Considered "..\Assemblies\Syncfusion.Compression.Base.winmd", but it didn't exist.
          Considered "..\Assemblies\Syncfusion.Compression.Base.dll", but it didn't exist.
          Considered "..\Assemblies\Syncfusion.Compression.Base.exe", but it didn't exist.
  Primary reference "Syncfusion.Pdf.Base, Version=20.2460.0.36, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL".
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2413,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "Syncfusion.Pdf.Base, Version=20.2460.0.36, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Core\Vcc.Nolvus.Core.csproj]
          For SearchPath "{TargetFrameworkDirectory}".
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Syncfusion.Pdf.Base.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Facades\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Facades\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Facades\Syncfusion.Pdf.Base.exe", but it didn't exist.
          For SearchPath "{AssemblyFoldersFromConfig:C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\AssemblyFolders.config,v4.8}".
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\SqlDb\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\SqlDb\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\SqlDb\Syncfusion.Pdf.Base.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\PublicAssemblies\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\PublicAssemblies\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\PublicAssemblies\Syncfusion.Pdf.Base.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.5\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.5\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.5\Syncfusion.Pdf.Base.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.0\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.0\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.0\Syncfusion.Pdf.Base.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v2.0\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v2.0\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v2.0\Syncfusion.Pdf.Base.exe", but it didn't exist.
          For SearchPath "{Registry:Software\Microsoft\.NETFramework,v4.8,AssemblyFoldersEx}".
          Considered AssemblyFoldersEx locations.
          For SearchPath "{AssemblyFolders}".
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\Syncfusion.Pdf.Base.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\Syncfusion.Pdf.Base.exe", but it didn't exist.
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\Syncfusion.Pdf.Base.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\Syncfusion.Pdf.Base.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\WiX Toolset v3.14\SDK\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\WiX Toolset v3.14\SDK\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\WiX Toolset v3.14\SDK\Syncfusion.Pdf.Base.exe", but it didn't exist.
          For SearchPath "{GAC}".
          Considered "Syncfusion.Pdf.Base, Version=20.2460.0.36, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL", which was not found in the GAC.
          For SearchPath "{RawFileName}".
          Considered treating "Syncfusion.Pdf.Base, Version=20.2460.0.36, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL" as a file name, but it didn't exist.
          For SearchPath "..\Assemblies\".
          Considered "..\Assemblies\Syncfusion.Pdf.Base.winmd", but it didn't exist.
          Considered "..\Assemblies\Syncfusion.Pdf.Base.dll", but it didn't exist.
          Considered "..\Assemblies\Syncfusion.Pdf.Base.exe", but it didn't exist.
CoreResGen:
  Processing resource file "Frames\DashboardFrame.resx" into "obj\Release\Vcc.Nolvus.Core.Frames.DashboardFrame.resources".
CoreCompile:
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\Roslyn\csc.exe /noconfig /nowarn:1701,1702 /fullpaths /nostdlib+ /errorreport:prompt /warn:4 /define:TRACE /highentropyva+ /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Microsoft.CSharp.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\mscorlib.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Core.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Data.DataSetExtensions.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Data.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Drawing.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Net.Http.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Windows.Forms.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Xml.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Xml.Linq.dll" /reference:D:\a\NolvusDashboard\NolvusDashboard\Assemblies\Vcc.Nolvus.Api.Installer.dll /reference:D:\a\NolvusDashboard\NolvusDashboard\packages\ZetaLongPaths.1.0.17\lib\net452-full\ZetaLongPaths.dll /debug:pdbonly /filealign:512 /optimize+ /out:obj\Release\Vcc.Nolvus.Core.dll /subsystemversion:6.00 /resource:obj\Release\Vcc.Nolvus.Core.Frames.DashboardFrame.resources /target:library /utf8output /langversion:7.3 Enums\Enums.cs Error\FaultyMod.cs Events\Browser.cs Events\Files.cs Events\Package.cs Events\Window.cs Frames\DashboardFrame.cs Frames\DashboardFrame.Designer.cs Frames\FrameParameters.cs Interfaces\IBrowserInstance.cs Interfaces\ICategory.cs Interfaces\IDashboard.cs Interfaces\IDashboardFrame.cs Interfaces\IErrorHandler.cs Interfaces\IFileService.cs Interfaces\IFolderService.cs Interfaces\IGameService.cs Interfaces\IGlobalsService.cs Interfaces\IInstallableElement.cs Interfaces\IInstanceOptions.cs Interfaces\IInstancePerformance.cs Interfaces\IInstanceService.cs Interfaces\IInstanceSettings.cs Interfaces\IInstanceStatus.cs Interfaces\ILibService.cs Interfaces\ILogService.cs Interfaces\IMod.cs Interfaces\ICheckerService.cs Interfaces\IModOrganizer.cs Interfaces\IModsContainer.cs Interfaces\IMOElement.cs Interfaces\IPackageService.cs Interfaces\IProgressQueue.cs Interfaces\IReportService.cs Interfaces\ISettingsService.cs Interfaces\INolvusInstance.cs Interfaces\IShellLink.cs Interfaces\ISoftware.cs Interfaces\ISoftwareProvider.cs Interfaces\IUpdaterService.cs Misc\ModObject.cs Misc\Statics.cs Properties\AssemblyInfo.cs Interfaces\INolvusService.cs Services\ServiceSingleton.cs "obj\Release\.NETFramework,Version=v4.8.AssemblyAttributes.cs"
D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Core\Interfaces\IReportService.cs(7,7): error CS0246: The type or namespace name 'Syncfusion' could not be found (are you missing a using directive or an assembly reference?) [D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Core\Vcc.Nolvus.Core.csproj]
D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Core\Interfaces\IReportService.cs(15,14): error CS0246: The type or namespace name 'PdfDocument' could not be found (are you missing a using directive or an assembly reference?) [D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Core\Vcc.Nolvus.Core.csproj]
  CompilerServer: server - server processed compilation - Vcc.Nolvus.Core
Done Building Project "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Core\Vcc.Nolvus.Core.csproj" (default targets) -- FAILED.
Done Building Project "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Core\Vcc.Nolvus.Core.csproj.metaproj" (default targets) -- FAILED.
Done Building Project "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Components\Vcc.Nolvus.Components.csproj.metaproj" (default targets) -- FAILED.
Project "D:\a\NolvusDashboard\NolvusDashboard\Nolvus Dashboard.sln" (1) is building "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.NexusApi\Vcc.Nolvus.NexusApi.csproj" (8) on node 1 (default targets).
PrepareForBuild:
  Creating directory "obj\Release\".
ResolveAssemblyReferences:
  Consider app.config remapping of assembly "System.Net.Http, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.1.1.3" [] to Version "4.2.0.0" [C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Net.Http.dll] to solve conflict and get rid of warning.
CoreCompile:
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\Roslyn\csc.exe /noconfig /nowarn:1701,1702 /fullpaths /nostdlib+ /errorreport:prompt /warn:4 /define:TRACE /highentropyva+ /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Microsoft.CSharp.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\mscorlib.dll" /reference:D:\a\NolvusDashboard\NolvusDashboard\packages\Newtonsoft.Json.13.0.1\lib\net45\Newtonsoft.Json.dll /reference:D:\a\NolvusDashboard\NolvusDashboard\packages\RestEase.1.5.0\lib\net45\RestEase.dll /reference:D:\a\NolvusDashboard\NolvusDashboard\packages\Serilog.2.10.1-dev-01249\lib\net46\Serilog.dll /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Core.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Data.DataSetExtensions.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Data.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Net.Http.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Runtime.Serialization.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Web.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Xml.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Xml.Linq.dll" /debug:pdbonly /filealign:512 /optimize+ /out:obj\Release\Vcc.Nolvus.NexusApi.dll /subsystemversion:6.00 /target:library /utf8output /langversion:7.3 ApiManager.cs HttpLoggingHandler.cs INexusAPI.cs IRestEaseClientFactory.cs MyRequestQueryParamSerializer.cs MyResponseDeserializer.cs Properties\AssemblyInfo.cs RateLimits.cs Responses\ColorScheme.cs Responses\Endorsement.cs Responses\EndorsementStatus.cs Responses\FileCategory.cs Responses\Game.cs Responses\GameCategory.cs Responses\Mod.cs Responses\ModFile.cs Responses\ModFileDownloadLink.cs Responses\ModFileList.cs Responses\ModFileUpdate.cs Responses\ModFileWithHash.cs Responses\ModHashResult.cs Responses\ModStatus.cs Responses\ModUpdate.cs Responses\User.cs Responses\UserEndorsement.cs Responses\UserTrackedMod.cs Responses\Validate.cs RestEaseClientFactory.cs Throttle.cs "obj\Release\.NETFramework,Version=v4.8.AssemblyAttributes.cs"
  CompilerServer: server - server processed compilation - Vcc.Nolvus.NexusApi
_CopyFilesMarkedCopyLocal:
  Copying file from "D:\a\NolvusDashboard\NolvusDashboard\packages\RestEase.1.5.0\lib\net45\RestEase.dll" to "D:\a\NolvusDashboard\NolvusDashboard\Assemblies\RestEase.dll".
  Copying file from "D:\a\NolvusDashboard\NolvusDashboard\packages\Serilog.2.10.1-dev-01249\lib\net46\Serilog.dll" to "D:\a\NolvusDashboard\NolvusDashboard\Assemblies\Serilog.dll".
  Copying file from "D:\a\NolvusDashboard\NolvusDashboard\packages\RestEase.1.5.0\lib\net45\RestEase.xml" to "D:\a\NolvusDashboard\NolvusDashboard\Assemblies\RestEase.xml".
  Copying file from "D:\a\NolvusDashboard\NolvusDashboard\packages\Serilog.2.10.1-dev-01249\lib\net46\Serilog.xml" to "D:\a\NolvusDashboard\NolvusDashboard\Assemblies\Serilog.xml".
  Creating "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.NexusApi\obj\Release\Vcc.Nolv.58516DD6.Up2Date" because "AlwaysCreate" was specified.
  Touching "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.NexusApi\obj\Release\Vcc.Nolv.58516DD6.Up2Date".
_CopyAppConfigFile:
  Copying file from "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.NexusApi\app.config" to "D:\a\NolvusDashboard\NolvusDashboard\Assemblies\Vcc.Nolvus.NexusApi.dll.config".
CopyFilesToOutputDirectory:
  Copying file from "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.NexusApi\obj\Release\Vcc.Nolvus.NexusApi.dll" to "D:\a\NolvusDashboard\NolvusDashboard\Assemblies\Vcc.Nolvus.NexusApi.dll".
  Vcc.Nolvus.NexusApi -> D:\a\NolvusDashboard\NolvusDashboard\Assemblies\Vcc.Nolvus.NexusApi.dll
  Copying file from "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.NexusApi\obj\Release\Vcc.Nolvus.NexusApi.pdb" to "D:\a\NolvusDashboard\NolvusDashboard\Assemblies\Vcc.Nolvus.NexusApi.pdb".
Done Building Project "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.NexusApi\Vcc.Nolvus.NexusApi.csproj" (default targets).
Project "D:\a\NolvusDashboard\NolvusDashboard\Nolvus Dashboard.sln" (1) is building "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Launcher\Vcc.Nolvus.Launcher.csproj" (14) on node 1 (default targets).
PrepareForBuild:
  Creating directory "..\Vcc.Nolvus.Dashboard\bin\".
  Creating directory "obj\Release\".
CoreCompile:
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\Roslyn\csc.exe /noconfig /nowarn:1701,1702 /fullpaths /nostdlib+ /platform:anycpu32bitpreferred /errorreport:prompt /warn:4 /define:TRACE /highentropyva+ /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\Microsoft.CSharp.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\mscorlib.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Core.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Data.DataSetExtensions.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Data.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Net.Http.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Xml.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Xml.Linq.dll" /debug:pdbonly /filealign:512 /optimize+ /out:obj\Release\NolvusLauncher.exe /subsystemversion:6.00 /target:exe /utf8output /win32icon:nolvus-ico.ico /langversion:7.3 Program.cs Properties\AssemblyInfo.cs "obj\Release\.NETFramework,Version=v4.8.AssemblyAttributes.cs"
  CompilerServer: server - server processed compilation - Vcc.Nolvus.Launcher
_CopyAppConfigFile:
  Copying file from "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Launcher\App.config" to "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Dashboard\bin\NolvusLauncher.exe.config".
CopyFilesToOutputDirectory:
  Copying file from "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Launcher\obj\Release\NolvusLauncher.exe" to "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Dashboard\bin\NolvusLauncher.exe".
  Vcc.Nolvus.Launcher -> D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Dashboard\bin\NolvusLauncher.exe
  Copying file from "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Launcher\obj\Release\NolvusLauncher.pdb" to "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Dashboard\bin\NolvusLauncher.pdb".
Done Building Project "D:\a\NolvusDashboard\NolvusDashboard\Vcc.Nolvus.Launcher\Vcc.Nolvus.Launcher.csproj" (default targets).
Done Building Project "D:\a\NolvusDashboard\NolvusDashboard\Nolvus Dashboard.sln" (default targets) -- FAILED.

Build FAILED.
```

**Comment by andreluizgaspar** [comment]
Hey! Thx @addy419 for the work!
Sorry for abandoning the project...
After I was banned from their Discord (my bad, sharing my premium Nexus API keys and stuff) I stopped working on this...
The Syncfusion dlls are proprietary... Vektor downloads those during installation and stuff...
Feel free to open a PR and add a Dockerfile if you want to... that would help ppl compile the project on Linux (I used to use a VM to work on the launcher).
With the release of the Oblivion Remaster I got back into gaming and Skyrim modpacks... mostly new Wabbajack modpacks tho... maybe I can check the new release of Nolvus and how that plays on Linux (Wabbajack is also not perfect, lots of crashes).
Thx again for the interest, and sorry for the lack of communication.
Being Wabbajack modspacks or Nolvus, theres lots of work to still do...

**Comment by addy419** [comment]
Hey @TheAndreLuiz, I don't mind. I have started using and contributing to the Gate to Sovengarde modlist. And while I didn't get banned from the Nolvus discord, I do not want to contribute anything from my side to this project. The maintainers have been really hostile on the Discord when I mentioned getting it to work on Linux. They did not want me contributing to it, and even accused me of trying to get it working behind their backs. 

TL;DR I do not want to be associated with the Nolvus project or the community in any manner. I am happy with contributing to projects that aren't hostile to Linux users.
